### ISSUE TITLE
Behavioral Code - Analysis Plan

### ISSUE THREAD

**Issue by d-robles** [APPLabUofA/Skateboard#14]
Goal: compare target RT by plotting facing in/facing out by preference type.

**Comment by kylemath** [comment]
why don't you start with this one first when you want to do some analysis, I think it should be pretty similar to what we have now, remember to make a new branch, call it something BehRegGoof or something, and do the work in there

**Comment by d-robles** [comment]
Sounds great. @kylemath

**Comment by d-robles** [comment]
![image](https://user-images.githubusercontent.com/38795068/57004718-53fb0a80-6b8e-11e9-8664-78f9c5955e4f.png)
@kylemath

**Comment by kylemath** [comment]
really cool,
but hard to see the effects that we care about (preferred vs non-preferred)

can you organize these differently so that preferred and non-preferred are
beside each other on one axis, and facing in an facing out are on different
axes.

so top left graph would be

facing in
preferred stance vs. non-preferred stance

**Comment by d-robles** [comment]
Alright

**Comment by d-robles** [comment]
![image](https://user-images.githubusercontent.com/38795068/57006048-a93d1900-6b9a-11e9-8711-73fbd5f0cec8.png)
@kylemath

**Comment by kylemath** [comment]
Awesome !

**Comment by d-robles** [comment]
@kylemath 

![image](https://user-images.githubusercontent.com/38795068/60550185-6c011e80-9ce4-11e9-91e5-fbb5f5fa6b3e.png)

I plotted accuracy by both facing orientation and preferred stance. 

Looking at it by facing orientation (upper plot) it seems that riding by preferred stance marginally brings up accuracy. 

Is there any other way you want me to look at the accuracy plots? 

The code is up to date on github case you want to go over what I did.
